charts/feature-cluster-metrics/templates/_node_exporter.alloy.tpl

```diff
@@ -58,7 +58,9 @@ node_exporter.scrape "metrics" {
 prometheus.relabel "node_exporter" {
   max_cache_size = {{ (index .Values "node-exporter").maxCacheSize | default .Values.global.maxCacheSize | int }}
 
+{{- if (index .Values "node-exporter").extraMetricProcessingRules }}
   {{(index .Values "node-exporter").extraMetricProcessingRules}}
```

Comment:
Only this one is missing `| indent 2`, not sure if that matters

Reply:
Whoa, expert catch! Thanks